packages/flutter/lib/src/foundation/change_notifier.dart

```diff
@@ -93,18 +94,25 @@ abstract class ValueListenable<T> extends Listenable {
   T get value;
 }
 
+class _ListenerEntry extends LinkedListEntry<_ListenerEntry> {
```

Comment:
This probably deserves a comment explaining why we have this wrapper object instead of just using the `listener` directly.

Reply:
It is required by LinkedList. We cannot do `LinkedList<VoidCallback>`.

We could change `LinkedList<_ListenerEntry>` by `DoubleLinkedQueue<VoidCallback>` maybe?

Reply:
Ah, I see. I didn't look at the interface for LinkedList. Never mind.